_includes/registration.html

```diff
@@ -34,6 +34,18 @@ <h3 class="center">Workshop Pass</h3>
         <a class="btn btn-success register" href="https://qkt.io/satRdayjoburg2020" target="BLANK">Buy Workshop Pass</a>
     </div>
 
+    <h3 class="center">Scholarship Application</h3>
+
+    <p>Thanks to funding from the R Consortium, we are able to offer a limited number of scholarships for women to attend the <b>R package development workshop</b> and the<b> satRday conference</b>. Due to time constraints, scholarships are open to women living in Africa who are eligible for visa-free access to South Africa (including women living in South Africa itself!).</p>
```

Comment:
I think remove the "Due to time constraints" and just start with "Scholarships are open...".

Also, "are open to women living in South Africa or elsewhere in Africa with visa-free access to South Africa.